rubble/src/link/ad_structure.rs

```diff
@@ -50,6 +50,12 @@ pub enum AdStructure<'a> {
     /// Sets the shortened device name.
     ShortenedLocalName(&'a str),
 
+    /// Set manufacturer specific data
+    ManufacturerSpecificData {
+        company_identifier: u16,
```

Comment:
Can you use `CompanyId` here? We already had some of the logic around company identifiers in place.

Reply:
sure